resolwe_bio/processes/expression/featureCounts.yml

```diff
@@ -14,7 +14,7 @@
       cores: 10
       network: true
   data_name: "{{ alignment.aligned_reads|sample_name|default('?') }}"
-  version: 2.4.2
+  version: 2.4.3
```

Comment:
I think this warrants a minor version bump, 2.5.0. Last digits are usually meant for patches.